Lib/unittest/mock.py

```diff
@@ -337,13 +337,19 @@ def __contains__(self, value):
 
         for i in range(0, len_self - len_value + 1):
             sub_list = self[i:i+len_value]
-            if sub_list == value:
+            if value == sub_list:
                 return True
         return False
 
     def __repr__(self):
         return pprint.pformat(list(self))
 
+    def __eq__(self, other):
+        self_list = list(self)
+        other_list = list(other)
+        # checking equality both directions is necessary for ANY to work
+        return self_list == other_list or other_list == self_list
```

Comment:
This `__eq__` doing similar check can be removed since `_CallList` contains `call` objects and there is `_Call.__eq__` that does `(self_params == other_params or other_params == self_params)`. I am not seeing any test failures on removing this, feel free to point me to tests if any.

Reply:
You're totally right. Embarrassed I missed that one. Thanks.